### ISSUE TITLE
QR code scanner in a small div or component

### ISSUE THREAD

**Issue by ChandAbdullah** [capacitor-community/barcode-scanner#108]
I Need the QR code like the below example

<img width="213" alt="Screenshot 2022-03-10 at 4 29 40 PM" src="https://user-images.githubusercontent.com/64844413/157653071-e30b239a-7d55-4d9b-8183-295aae4e6620.png">

**Comment by JanMisker** [comment]
The camera view itself will always be fullscreen. The webview background is made transparent to show it. But you can just put a non-transparent div in your app to block parts of the camera view.

**Comment by thegnuu** [comment]
@ChandAbdullah did this work for you? We know that currently codes can be scanned outside of the div you are defining, unfortunately this is kinda hard to achieve with the current state of the project. We are working on a solution for this problem, but it might take a while and it will require some work on your side as well once it is done.

**Comment by thegnuu** [comment]
@ChandAbdullah since I have not heard anything I will close this issue. If you still having troubles feel free to reopen it!

**Comment by j4vs** [comment]
Similar problem, i'm making a view for a price checker. So i need just the view camera on the top of the screen so i can show the products on the bottom of the screen. The problem is if put a div in front of the camera view, the top part doesn't recognize de code bar because of the scan area



![demo](https://user-images.githubusercontent.com/51412848/178576336-e7ee0578-1943-466f-bcc0-4d52acdb2378.jpeg)

**Comment by thegnuu** [comment]
Yeah, this is currently not possible. I am working on a version which would allow things like this, but unfortunately I am not able to provide a date when it might be ready.

The iOS version is nearly ready, but the Android version might take a while if I have to do it all by myself...

Sorry for not being able to provide you better news ;)

**Comment by yucehasan** [comment]
Hello @thegnuu . Is there any update on this? Will we be able to use the plugin as shown in screenshots?

**Comment by thegnuu** [comment]
We will yes, someone is currently working on the Android implementation of MLKit and as far as I am informed the results are quite promising there as well. With this you will be able to get the x, y, width and height values of the scanned barcode, therefore you will be able to only accept codes scanned in your defined area.

Unfortunately there is still no timeline...

**Comment by fozzarelo** [comment]
+1

**Comment by kikaocg** [comment]
+1
